### PR TITLE
Switched to TestAttributes

### PR DESCRIPTION
**Pull request by smudge202**
Switched us over to TestAttributes. Nice job, @herecydev. I see that Traits are not being detected, but until we start pushing in new test types that isn't an issue.

I believe this replaces @danielcirket's pr #12?

### CODE REVIEW

**Comment by ghost** [comment]
For completeness, did you raise an issue about the traits on the TA repo?

File: tests/Component.Tests/project.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
 		"Components": "1.0.0-*",
 		"FluentAssertions": "4.0.0",
 		"Microsoft.Framework.DependencyInjection": "1.0.0-beta8",
+		"TestAttributes": "2.0.0-rc1",
 		"xunit": "2.1.0",
```
Comment (ghost) [inline tests/Component.Tests/project.json:13]: Wondering whether we need to explicitly state the xunit dependencies? I wouldn't have said so.

**Comment by ghost** [comment]
Other than the above :shipit:

File: tests/Component.Tests/project.json (hunk 1 above)
Comment (smudge202) [inline tests/Component.Tests/project.json:13]: No, we don't. That's half the point of this new system; dependency versions are not tracked.

**Comment by smudge202** [comment]
Issue raised:  herecydev/TestAttributes#36

**Pull request merged**